sklearn/cluster/_optics.py

```diff
@@ -753,11 +753,11 @@ def _xi_cluster(reachability_plot, predecessor_plot, ordering, xi, min_samples,
 
     Parameters
     ----------
-    reachability_plot : array, shape (n_samples)
+    reachability_plot : array of shape (n_samples,)
```

Comment:
Same here (if it can be a `list`, use `array-like`, otherwise `ndarray`).

Reply:
ok